29.md

```diff
@@ -0,0 +1,41 @@
+NIP029
+======
+
+Image Metadata
+--------------
+
+Image metadata is additional information associated with urls in notes that
```

Comment:
```suggestion
`imeta` is a tag for adding media attachments to events. Typically `imeta` tags match URLs in the event content, but `imeta` tags with no matching URLs SHOULD still be displayed with the event (such as underneath the post). Clients may replace imeta URLs with rich previews. `imeta` tags contain extra information about the media attachment, which
```

Reply:
Is the idea to include videos, gifs, and PDFs in this?

Reply:
Yes, why not? Clients shouldn't display previews for any file type they don't want to. They can display a generic "file download" component.

Reply:
@staab I updated my suggested text here, but GitHub doesn't give me any way to commit it.